_health-care/_js/utils/options-for-select.js

```diff
@@ -237,6 +237,18 @@ const maritalStatuses = [
   'Divorced'
 ];
 
+const branchesServed = [
+  'Army',
+  'Airforce',
+  'Marine',
+  'Navy'
+];
+
+const dischargeTypes = [
```

Comment:
Discharge types in the original form are:

```
HONORABLE
GENERAL
OTHER THAN HONORABLE
BAD CONDUCT
DISHONORABLE
UNDESIRABLE
```

and Service Branches are (_I've left them in XML form since there's a difference between user text and in-XML text:

```
                    <items>
                      <text>Select Last Service Branch</text>
                      <text>AIR FORCE</text>
                      <text>ARMY</text>
                      <text>COAST GUARD</text>
                      <text>MARINE CORPS</text>
                      <text>MERCHANT SEAMAN</text>
                      <text>NAVY</text>
                      <text>NOAA</text>
                      <text>USPHS</text>
                      <text>FILIPINO COMMONWEALTH ARMY</text>
                      <text>FILIPINO GUERILLA FORCES</text>
                      <text>FILIPINO NEW SCOUT</text>
                      <text>FILIPINO OLD SCOUT</text>
                      <text>OTHER</text>
                    </items>
                    <items save="1" presence="hidden">
                      <text> </text>
                      <text>AIR FORCE</text>
                      <text>ARMY</text>
                      <text>COAST GUARD</text>
                      <text>MARINE CORPS</text>
                      <text>MERCHANT SEAMAN</text>
                      <text>NAVY</text>
                      <text>NOAA</text>
                      <text>USPHS</text>
                      <text>F.COMMONWEALTH</text>
                      <text>F.GUERILLA</text>
                      <text>F.SCOUTS NEW</text>
                      <text>F.SCOUTS OLD</text>
                      <text>OTHER</text>
                    </items>
```